packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -142,7 +142,10 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
     TextHeightBehavior? textHeightBehavior,
     TextWidthBasis textWidthBasis = TextWidthBasis.parent,
     String obscuringCharacter = '•',
-    bool obscureText = false,
+    @Deprecated(
+      'Use obscureTextBehavior instead. '
+      'This feature was deprecated after v2.6.0-0.0.pre.',
+    )
```

Comment:
I think you meant to keep `obscureText` and add the Deprecated flag above it?

Reply:
yes,now it does not a break change.
i found many error files,but i does not edit it

Reply:
This still looks like a problem here.  There is no obscureTextBehavior, and obscureText is removed.  obscureText should remain and obscureTextBehavior should be added, right?